pwa/src/styles/main.css

```diff
@@ -236,3 +236,24 @@ a:hover {
 .utrecht-button:disabled:hover {
     cursor: not-allowed;
 }
+
+.circle {
+    display: flex;
+    justify-content: center;
+    font-size: 12px;
+    border-radius: 2px;
+    padding: 6px 10px;
+    height: 30px;
+    width: 65px;
+}
+
+.circle.false {
+    color: #818182;
+    background-color: #e6e6e6;
+}
+
+.circle.true {
+    color: #004085;
+    background-color: #cce5ff;
+
+}
```

Comment:
We're currently using this type of label in three places: `AttributeTable`, `DashboardLogsTable` and `DashboardCard`. For each of these implementations, we're building the label from scratch (meaning we're duplicating the `html`, `ts` and `css` code three times).

I suggest making one component named (e.g.) `LabelWithBackground`, we can style this component once. When we have this component, we can then include it in `AttributeTable`, `DashboardLogsTable` and `DashboardCard`.

We could style the specific implementations with a `type` prop on the component, something like:

```
interface LabelWithBackgroundProps {
  label: string;
  type: "success" | "danger" | "primary" | "secondary";
}

const LabelWithBackground: React.FC<LabelWithBackgroundProps> = () => {
  return <></>; // create the component here
};
```

This way, we could implement the component as such:

```
<LabelWithBackground label="optional" type="secondary" />
```

and we therefore do not actually need any of the styling logic **within the implementation component** but we can keep all this logic separated.